src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -20,7 +20,7 @@
       "version": "4.0.0-rc2-23907",
       "exclude": "Compile"
     },
-    "MSBuild": "0.1.0-preview-00017",
+    "MSBuild": "0.1.0-preview-00018",
```

Comment:
Yes, this was a very old PR containing a fix enabling MSBuild to self-host on core clr and not on Mono. You can close this PR.

However, I also made this other PR (#763) that enables MSBuild to have p2p versioning inside its nuspecs. Once I have time to update the PR it would be nice if that one got in :)